### DISCUSSION TITLE
Custom legend Items

### DISCUSSION THREAD

**Discussion by marmanillo** [apexcharts/apexcharts.js#3382] [Q&A]
I am trying to use this option but it doesn't show me the legend that I want to visualize.

How does this customLegendItems work correctly?

**Comment by ichlasdev** [comment]
make sure you enabled legend to be shown
`legend: {
    show: true
  }`

here some example of how to use it
`legend: {
    show: true,
    customLegendItems: ['>3.000', '>5.000'],
    markers: {
        fillColors: ['#FFC107', '#DC3545'],
    }
}`
